bin/clojerl script fails in Linux

**Describe the bug**
When using a Debian based Linux distribution running the `bin/clojerl` script with options results in the following error:
```
$ bin/clojerl -r
bin/clojerl: 64: bin/clojerl: Bad substitution
make: *** [repl] Error 2
```

**To Reproduce**
See above.

**Expected behavior**
Process options correctly

**Actual behavior**
See above.

**Context**
 - `clojerl` version (tag/sha): `0.6.0-25-g5f9b53e`

--- a/bin/clojerl
+++ b/bin/clojerl
@@ -61,9 +61,8 @@ do
             ;;
         *)
             # check if first char is dash
-            first_char=${key:0:1}
-            case $first_char in
-                -)
+            case $key in
+                -*)
                     break;
                     ;;
                 *)

